manifests/etcdquorumguard/deployment.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: etcd-quorum-guard
+  namespace: {{.TargetNamespace}}
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      k8s-app: etcd-quorum-guard
+  strategy:
+    rollingUpdate:
+      maxSurge: 0
+      maxUnavailable: 1
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        name: etcd-quorum-guard
+        k8s-app: etcd-quorum-guard
+    spec:
+      hostNetwork: true
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchExpressions:
+              - key: k8s-app
+                operator: In
+                values:
+                - "etcd-quorum-guard"
+            topologyKey: kubernetes.io/hostname
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
+      priorityClassName: "system-cluster-critical"
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        effect: NoSchedule
+        operator: Exists
+      - key: node.kubernetes.io/memory-pressure
+        effect: NoSchedule
+        operator: Exists
+      - key: node.kubernetes.io/disk-pressure
+        effect: NoSchedule
+        operator: Exists
+      - key: node.kubernetes.io/not-ready
+        effect: NoExecute
+        operator: Exists
+      - key: node.kubernetes.io/unreachable
+        effect: NoExecute
+        operator: Exists
+      - key: node.kubernetes.io/unschedulable
+        effect: NoExecute
+        operator: Exists
+      - key: node-role.kubernetes.io/etcd
+        operator: Exists
+        effect: NoSchedule
+      containers:
+      - image: "{{.Images.EtcdQuorumGuardImage}}"
+        imagePullPolicy: IfNotPresent
+        name: etcd-quorum-guard-container
+        volumeMounts:
+        - mountPath: "/etc/ssl/certs/etcd"
+          name: etcd-metric-client
+          readOnly: true
+        - mountPath: "/var/run/etcd-metric-serving-ca"
+          name: etcd-metric-serving-ca
+          readOnly: true
+        command:
+        - "/bin/sh"
+        args:
+        - "-c"
+        - |
+          declare -r croot=/etc/ssl/certs/etcd
+          set -x
+          declare -r health_endpoint="https://127.0.0.1:9979/health"
+          declare -r cert="$croot/tls.crt"
+          declare -r key="$croot/tls.key"
+          declare -r cacert="/var/run/secrets/kubernetes.io/serviceaccount/etcd-metric-serving-ca.crt"
+          ls -lR "$croot"
```

Comment:
`etcd-metric-serving-ca` is  a ConfigMap vs Secret

https://github.com/openshift/machine-config-operator/blob/f4d79247db439ae08e2f5e17cf0347c94998e94d/pkg/operator/sync.go#L375-L378